vercel/turbo
# Move fs cache location.

### Description

Moves the file system cache to `.turbo` as discussed in #842.

### Testing Instructions

Find your favorite Turborepo of choice and run a `turbo` command twice. You should miss cache on the first run and hit cache on the second.

CLOSES TURBO-3130
